### PR TITLE
Update `.editorconfig` rules

### PR DESCRIPTION
**Pull request by msasikanth**
- Update `.editorconfig`
- Run formatter on project
- Update CHANGELOG

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -12,6 +9,7 @@ A clear and concise description of what the bug is.
 
 **To Reproduce**
 Steps to reproduce the behavior:
+
```
Comment (janhavisinghh) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: Can we remove this extra line please? Or is this intentional?

**Review by janhavisinghh — CHANGES_REQUESTED** [review]
LGTM. Left a comment PTAL?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (msasikanth) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: It's intentional, it's made by the formatter.

**Review by janhavisinghh — APPROVED** [review]
LGTM

**Pull request merged**